src/alembic/versions/20251016_1000_7e5b5cf430ff_destructive_migration_filesystem_and_.py

```
@@ -0,0 +1,178 @@
+                WHERE datname = current_database()
+                AND pid <> pg_backend_pid()
+            """
+                )
+            )
+            # Drop and recreate schema
+            connection.execute(text("DROP SCHEMA public CASCADE"))
+            connection.execute(text("CREATE SCHEMA public"))
+            connection.execute(text("GRANT ALL ON SCHEMA public TO public"))
+
+        logger.log("DATABASE", "Database reset complete")
+        return True
```

Ruff (0.14.0)

Consider moving this statement to an `else` block

(TRY300)